hiring/README.md

```diff
@@ -0,0 +1,26 @@
+Digital Publishing Whistlestop Tour
```

Comment:
I dont like this name but can't think of anything better - suggestions welcome!

Reply:
outline, overview, synopsis, digest,summary ??